perma-payments/Pipfile

```diff
@@ -5,7 +5,7 @@ verify_ssl = true
 
 [packages]
 
-django = "~=1.11.21"
+django = "~=1.11.22"
```

Comment:
FWIW, this change shouldn't have been necessary: 1.11.22 matches "~=1.11.21", so just running `pipenv lock` would have had the same effect.